src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemConf.scala

```diff
@@ -29,6 +29,10 @@ class FileItemConf(args: Seq[String]) extends ScallopConf(args) {
 
   printedName = "easy-stage-file-item"
   version(s"$printedName v${Version()}")
+
+  editBuilder(_.setHelpWidth(100))
+  val NB_SPACE = "\u00A0" // &nbsp; won't work when calling for help on a console
```

Comment:
@jo-pol : `&nbsp;` is an [html-entity](http://www.w3schools.com/html/html_entities.asp). The help is plain text, so it doesn't make sense to expect `&nbsp;` to be replaced by the corresponding character.